superset-frontend/src/explore/components/DataTablesPane/DataTablesPane.test.tsx

```diff
@@ -0,0 +1,135 @@
+/**
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+import userEvent from '@testing-library/user-event';
+import React from 'react';
+import { render, screen, act } from 'spec/helpers/testing-library';
+import { DataTablesPane } from '.';
+
+let defaultProps = {
```

Comment:
If the props are not modified, you can change it to a `const` and remove `beforeEach`. If only some are being modified, you can override them inside each test.

Reply:
Same reason here: https://github.com/apache/superset/pull/13753#discussion_r600729737

Reply:
done: https://github.com/apache/superset/pull/13753#discussion_r600862661